src/main/java/org/vietabroader/view/MainView.java

```diff
@@ -30,9 +51,20 @@ private void initUI() {
         pack();
     }
 
+    private class DefaultPanel extends JPanel {
```

Comment:
I think it saves some duplicated code in setting up each panel. If it's not that helpful then I remove this class